ocw/management/commands/dumpstate.py

```diff
@@ -0,0 +1,10 @@
+from django.core.management.base import BaseCommand
+from ocw.lib.dump_state import dump_state
+
+
+class Command(BaseCommand):
```

Comment:
I would use a less generic name for this class.

Reply:
please check ALL other files in this folder :) I am not 100% sure but this is might be requirement from django how this "commands flow" is working . In any case no one except django itself suppose to create instances of this class so this one and only place where this name will be used